pkg/util/test/util.go

```diff
@@ -894,7 +899,7 @@ func validatePxPodsAnnotations(cluster *corev1.StorageCluster, timeout, interval
 			// This should validate the annotation cluster-autoscaler.kubernetes.io/safe-to-evict value,
 			// when node is Storage, annotation value should be set to false,
 			// when node is Storageless, annotation value should be set to true
-			if storageNode.Status.NodeAttributes.Storage != nil && *storageNode.Status.NodeAttributes.Storage != !safeToEvict {
+			if opVersion.GreaterThanOrEqual(OpVer23_10_3) && storageNode.Status.NodeAttributes.Storage != nil && *storageNode.Status.NodeAttributes.Storage != !safeToEvict {
```

Comment:
This is not gonna work, we should either find a different way to validate this or skip the whole thing all together for old operator and just print something like PX Operator version is too old and we do not support validating this

Reply:
Also next debug print after this statement will not be useful as the older PX Operators do not have `*storageNode.Status.NodeAttributes.Storage`

Reply:
Addressed the comments.